cnames_active.js

```diff
@@ -1936,6 +1936,7 @@ var cnames_active = {
   "minidrone": "mechazawa.github.io/minidrone-js",
   "minify": "minify-js.github.io",
   "minigrid": "henriquea.github.io/minigrid", // noCF? (don´t add this in a new PR)
+  "minigfm": "vercel.cdn.yt-blog.top", // noCF
```

Comment:
```suggestion
  "minigfm": "vercel.cdn.yt-blog.top", // noCF
  "minigrid": "henriquea.github.io/minigrid", // noCF? (don´t add this in a new PR)
```
As the CI error notes, your entry needs to be in alphabetical order